src/pages/StudentDashboardPage/DashboardComponents/CourseMediaCard.js

```diff
@@ -14,9 +14,7 @@ import React from "react";
 const useStyle = makeStyles((theme) => ({
   root: {
     marginTop: theme.spacing(1),
-    marginLeft: "10px",
-    marginRight: "18px",
-    minWidth: 300,
+    minWidth: 270,
```

Comment:
instead of having this we can define width according in `vw` units. That will fix the overall problems with different screen size.
Something like: `width: '80vw'`

Reply:
no, actually doing this will make cards something like this.
<img width="487" alt="Screenshot 2021-04-17 at 12 39 58" src="https://user-images.githubusercontent.com/66865329/115105186-d72fe980-9f7a-11eb-96a7-5d014c9b35d4.png">

Reply:
There is some bug, rightmost card's right margin is not getting reflected.

Reply:
@Abhishek-kumar09 Could you please help out in this bug?